### ISSUE TITLE
An internal error accurred while attempting to pertorm this command

### ISSUE THREAD

**Issue by nenohi** [dsdd/EnchantmentsPlusMinus#48]
If you attach a certain enchantment, an error will occur and you will not be able to attach it any more.

![image](https://user-images.githubusercontent.com/18223268/187181373-a5538e64-f67c-4dba-8848-c292275da063.png)

enchantment list
Efficiency 10
Fly
Experience 2
Fortune 7
Tokenblock 19
Unbreaking 20

Is there any way to fix this?

Minecraft server version
sipgot: 1.19

**Comment by nenohi** [comment]
server console log
```
[19:30:05] [Server thread/INFO]: xxnenohixx issued server command: /enchantments
[19:30:05] [Server thread/ERROR]: null
org.bukkit.command.CommandException: Unhandled exception executing command 'enchantments' in plugin EnchantmentsPlusMinus v1.9r-11
        at org.bukkit.command.PluginCommand.execute(PluginCommand.java:47) ~[spigot-api-1.19-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:149) ~[spigot-api-1.19-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.craftbukkit.v1_19_R1.CraftServer.dispatchCommand(CraftServer.java:831) ~[spigot-1.19-R0.1-SNAPSHOT.jar:3553-Spigot-14a2382-ef09464]
        at net.minecraft.server.network.PlayerConnection.handleCommand(PlayerConnection.java:2031) ~[spigot-1.19-R0.1-SNAPSHOT.jar:3553-Spigot-14a2382-ef09464]
        at net.minecraft.server.network.PlayerConnection.a(PlayerConnection.java:1834) ~[spigot-1.19-R0.1-SNAPSHOT.jar:3553-Spigot-14a2382-ef09464]
        at net.minecraft.network.protocol.game.ServerboundChatCommandPacket.a(SourceFile:37) ~[spigot-1.19-R0.1-SNAPSHOT.jar:3553-Spigot-14a2382-ef09464]
        at net.minecraft.network.protocol.game.ServerboundChatCommandPacket.a(SourceFile:13) ~[spigot-1.19-R0.1-SNAPSHOT.jar:3553-Spigot-14a2382-ef09464]
        at net.minecraft.network.protocol.PlayerConnectionUtils.lambda$0(PlayerConnectionUtils.java:31) ~[spigot-1.19-R0.1-SNAPSHOT.jar:3553-Spigot-14a2382-ef09464]
        at net.minecraft.server.TickTask.run(SourceFile:18) ~[spigot-1.19-R0.1-SNAPSHOT.jar:3553-Spigot-14a2382-ef09464]
        at net.minecraft.util.thread.IAsyncTaskHandler.d(SourceFile:157) ~[spigot-1.19-R0.1-SNAPSHOT.jar:3553-Spigot-14a2382-ef09464]
        at net.minecraft.util.thread.IAsyncTaskHandlerReentrant.d(SourceFile:23) ~[spigot-1.19-R0.1-SNAPSHOT.jar:3553-Spigot-14a2382-ef09464]
        at net.minecraft.server.MinecraftServer.b(MinecraftServer.java:1150) ~[spigot-1.19-R0.1-SNAPSHOT.jar:3553-Spigot-14a2382-ef09464]
        at net.minecraft.server.MinecraftServer.d(MinecraftServer.java:1) ~[spigot-1.19-R0.1-SNAPSHOT.jar:3553-Spigot-14a2382-ef09464]
        at net.minecraft.util.thread.IAsyncTaskHandler.x(SourceFile:131) ~[spigot-1.19-R0.1-SNAPSHOT.jar:3553-Spigot-14a2382-ef09464]
        at net.minecraft.server.MinecraftServer.bh(MinecraftServer.java:1129) ~[spigot-1.19-R0.1-SNAPSHOT.jar:3553-Spigot-14a2382-ef09464]
        at net.minecraft.server.MinecraftServer.x(MinecraftServer.java:1122) ~[spigot-1.19-R0.1-SNAPSHOT.jar:3553-Spigot-14a2382-ef09464]
        at net.minecraft.util.thread.IAsyncTaskHandler.c(SourceFile:140) ~[spigot-1.19-R0.1-SNAPSHOT.jar:3553-Spigot-14a2382-ef09464]
        at net.minecraft.server.MinecraftServer.w(MinecraftServer.java:1106) ~[spigot-1.19-R0.1-SNAPSHOT.jar:3553-Spigot-14a2382-ef09464]
        at net.minecraft.server.MinecraftServer.v(MinecraftServer.java:1014) ~[spigot-1.19-R0.1-SNAPSHOT.jar:3553-Spigot-14a2382-ef09464]
        at net.minecraft.server.MinecraftServer.lambda$0(MinecraftServer.java:291) ~[spigot-1.19-R0.1-SNAPSHOT.jar:3553-Spigot-14a2382-ef09464]
        at java.lang.Thread.run(Thread.java:833) [?:?]
Caused by: java.lang.ArrayIndexOutOfBoundsException: Index -1 out of bounds for length 8
        at org.vivi.eps.libs.NumberUtils.abbreviate(NumberUtils.java:18) ~[?:?]
        at org.vivi.eps.visual.EnchantGUI.add(EnchantGUI.java:188) ~[?:?]
        at org.vivi.eps.visual.EnchantGUI.loadInventory(EnchantGUI.java:117) ~[?:?]
        at org.vivi.eps.visual.EnchantGUI.openInventory(EnchantGUI.java:96) ~[?:?]
        at org.vivi.eps.command.EnchantsCommand.onCommand(EnchantsCommand.java:59) ~[?:?]
        at org.bukkit.command.PluginCommand.execute(PluginCommand.java:45) ~[spigot-api-1.19-R0.1-SNAPSHOT.jar:?]
        ... 20 more
```

**Comment by dsdd** [comment]
Fixed in 1.9r-12

**Comment by nenohi** [comment]
thx
